package.json

```diff
@@ -26,9 +26,9 @@
     "lint": "dotenv -e test.env -- npm run build && prettier --list-different \"**.ts\" && eslint --fix \"**.ts\"",
     "lint:shell": "shellcheck ./bin/* ./scripts/* ./.husky/pre-commit",
     "lint:staged": "lint-staged",
-    "test": "dotenv -e test.env -- npm run build:templates && dotenv -e test.env -- tap ./test",
+    "test": "dotenv -e test.env -- npm run build:templates && dotenv -e test.env -- vitest --run && dotenv -e test.env tap ./test/*.tap.ts",
```

Comment:
Should I introduce a test:watch that uses vitest's watch mode?

Reply:
That would be a cool feature, however the template generation component means I'd need to introduce chokidar at a minimum, *and* I'd have to contend with that setting colliding with `npm start` run concurrently. I'd rather not bite that off for now. But it's an interesting option for later, and I might implement it when working on my next big feature.